### DISCUSSION TITLE
Error faild to resolve import "vuefire"

### DISCUSSION THREAD

**Discussion by tkohlmannhuber** [vuejs/vuefire#1555] [Questions and Help]
Hi. 

I have a completly fresh Nuxt Project and i wanted to add vuefire for auth. but as soon as im installing the Module and adding the config im getting this error `Pre-transform error: Failed to resolve import "vuefire" from "virtual:nuxt:/Users/thomaskohlmannhuber/Sites/firebase-auth/firebase/.nuxt/vuefire-plugin.mjs". Does the file exist? (x1)`.
I really dont know how to fix this. 
Thx for helping

**Comment by fabian1606** [comment]
Hi, I have the same problem. Did you fix it?

**Comment by fabian1606** [comment]
I solved the issue by doing the steps in the [documentation](https://vuefire.vuejs.org/nuxt/getting-started.html) and doing yarn add vuefire (or npm i vuefire)
